CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+# v0.0.44
+
+## Announcing Akri v0.0.44!
+Akri v0.0.44 is a pre-release of Akri.
+
+To find out more about Akri, check out our [README](https://github.com/deislabs/akri/blob/main/README.md) and start [contributing](https://github.com/deislabs/akri/blob/main/docs/contributing.md) today!
+
+## New Features
+The v0.0.44 release introduces a number of significant improvements!
+
+* Enable Akri for armv7
+* Create separate Helm charts for releases (akri) and merges (akri-dev)
+* Parameterize Helm for udev beyond simple video scenario
+* Expand udev discovery to include subsystems
```

Comment:
This is beyond subsystems filtering support. Maybe change to "Expand udev discovery by supporting filtering by udev rules that look up the device hierarchy such as SUBSYSTEMS, ATTRIBUTES, DRIVERS, KERNELS, and TAGS"

Reply:
👍